README.md

```diff
@@ -1,2 +1,35 @@
 # tf-github
-Terraform code to create github resources
+
+Terraform code to manage GitHub repositories.
+Resources currently managed by this repository are:
+ * GitHub repositories
+ * Repository branch protections
+ * Repository opermissions
+ * Generic files for multiple repositories
+
+## CICD
+
+To deploy changes from this repository, we've installed a Terraform app to provide authentication for the Terraform actions performed on the GitHub resources.
+
+Two secrets have been created in the repository settings itself, which are needed to run the GitHub Actions:
```

Comment:
Might want to say "multiple" instead of "two" in case the number increases, there are three in the table